### DISCUSSION TITLE
Relative path prefix for next/image with remote images

### DISCUSSION THREAD

**Discussion by yeldiRium** [vercel/next.js#64652] [Ideas]
# Bug report

## Describe the bug

When `images.path` in `next.config.js` is set to `./_next/image`, and when using the `next/image` component with `src=https://img.someserver.com/path/to/image.jpg`, the resulting request made to the server is to `/_next/image/?url=...`

## To Reproduce

1. Set `images.path` in `next.config.js` to `./_next/image`
2. Set `images.domains` in `next.config.js` to '[ 'lorempixel.com' ]
3. Add a component to your site: `<NextImage src={ 'http://lorempixel.com/400/200/' } width={ 200 } />`
4. Start nextjs and open the site

## Expected behavior

As with the `assetPrefix`, I expect a relative path for the images to make the requested urls relative. So I expect the resulting request to be made to `./_next/image?url=...`.

What makes this even more confusing: The markup seems to be generated correctly. The dom contains on the generated img tag `data-src="./_next/image?url=http%3A%2F%2Florempixel.com%2F400%2F200%2F&w=828&q=75"` and `data-srcset="./_next/image?url=http%3A%2F%2Florempixel.com%2F400%2F200%2F&w=384&q=75 1x, ./_next/image?url=http%3A%2F%2Florempixel.com%2F400%2F200%2F&w=640&q=75 2x, ./_next/image?url=http%3A%2F%2Florempixel.com%2F400%2F200%2F&w=828&q=75 3x"`. The chrome network tab however show an outgoing request to `http://localhost:3000/_next/image/?url=http%3A%2F%2Florempixel.com%2F400%2F200%2F&w=384&q=75` (`http://localhost:3000` is where I'm running the server).

## System information

- OS: manjaro linux, macOS
- Browser: chrome
- Version of Next.js: 10.0.1
- Version of Node.js: 12.16.1, 14.14.0

**Comment by bigclap** [comment]
I use it for proxy remote images

**Comment by RacketyWater7** [comment]
Any update on this issue?

**Comment by JuniorTour** [comment]
The default behavior of `next/image` will add `/_next/image?url=` to any absolute URL of image `src`.

A workaround is use `imgix loader` and set path to `''`:
``` js
// next.config.js
module.exports = {
  images: {
    domains: ['yourServer.com'],
    path: '',
    loader: 'imgix',
  },
}
```

<hr />

From source code, add `root = '/_next/image?url=...'` to any absolute URL `src` is by design:
https://github.com/vercel/next.js/blob/f81a6a5f0fc3a33a51112d4d3261d431e704b0da/packages/next/client/image.tsx#L862

Why? 

It's a little strange, different from the HTML `<image src="">` behavior, makes users confused too.

**Comment by dharanchit** [comment]
Any update on this?
